### ISSUE TITLE
Error importing bmipy-render output on Windows

### ISSUE THREAD

**Issue by mdpiper** [csdms/bmi-python#10]
On Windows, an error is thrown when attempting to import from a file created by `bmipy-render`. To reproduce it, try these steps:

1. Clone the bmi-python repo and install *bmipy* from source.
1. From the command line (I used the Git Bash shell), use `bmipy-render` to create a new file for a MyBmi class:
```
$ bmipy-render MyBmi > mybmi.py
```
1. Try to import the class from the module:
```
$ python -c "from mybmi import MyBmi"
```

The result is an error, with message
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Users\Mark\projects\bmi-python\mybmi.py", line 539
    """
       ^
SyntaxError: (unicode error) 'utf-8' codec can't decode byte 0x93 in position 483: invalid start byte
```

**Comment by mdpiper** [comment]
The same error occurs using the conda-forge *bmipy* package.

**Comment by mdpiper** [comment]
Be sure to revert 5c9bc7604059a4cff473fa8de13d71149a263e23 when this issue is fixed.

**Comment by mcflugen** [comment]
Fixed with #45.
